modules/smartadserverBidAdapter.md

```diff
@@ -94,4 +94,41 @@ Please reach out to your Technical account manager for more information.
             }
         }]
     };
+```
+
+## Outstream Video
+
+```
+    var outstreamVideoAdUnit = {
+			code: 'test-div',
+			mediaTypes: {
+				video: {
+					context: 'outstream',
+					playerSize: [640, 480]
+				}
+			},
+			renderer: {
+				url: 'https://acdn.adnxs.com/video/outstream/ANOutstreamVideo.js',
+				render: function (bid) {
+					ANOutstreamVideo.renderAd({
+						targetId: bid.adUnitCode,
+						adResponse: bid
+					});
```

Comment:
I think an update is needed here.  In the documentation [here](https://docs.prebid.org/dev-docs/show-outstream-video-ads.html#renderers), it has the actual renderer call wrapped around a queue handler to ensure the called gets executed at the right time.

When I was testing with these exact params, I was getting an error during the rendering step that the `ANOustreamVideo` did not exist.  I think it just means this call here was happening too early before the script fully deployed.  Can you make an update here (just so the example is right)?

Reply:
Thanks for the doc, I fixed it.